### PR TITLE
upgrade version for dependant CFT modules

### PR DESCRIPTION
**Pull request by ericyz**

### CODE REVIEW

**Review by morgante — APPROVED** [review]

File: 1-org/envs/shared/versions.tf [hunk 1]
```diff
@@ -17,3 +17,16 @@
 terraform {
   required_version = ">= 0.13"
 }
+terraform {
+  required_version = ">=0.13, <0.14"
```
Comment (bharathkkb) [inline 1-org/envs/shared/versions.tf:21]: there seems to be conflicting `terraform` blocks with required_version. We should keep these to `required_version >= 0.13` to allow users flexibility

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @ericyz

File: 1-org/envs/shared/versions.tf (hunk 1 above)
Comment (bharathkkb) [reply 1-org/envs/shared/versions.tf:21]: this seems to be in other updated versions files as well

File: 1-org/envs/shared/versions.tf [hunk 2]
```diff
@@ -17,3 +17,16 @@
 terraform {
```
Comment (bharathkkb) [inline 1-org/envs/shared/versions.tf:17]: @ericyz we should remove this extraneous `terraform` block here and throughout

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**